CI reports publish_api_docs.sh not found.

Description:

Run bash ./publish_api_docs.sh
bash: ./publish_api_docs.sh: No such file or directory
Error: Process completed with exit code 127.

Due to script rename CI pipeline finished with error.

--- a/.github/workflows/publish_api_docs.yml
+++ b/.github/workflows/publish_api_docs.yml
@@ -20,7 +20,7 @@ jobs:
       run: sudo apt-get install qdoc-qt5 qtbase5-dev
 
     - name: Generating and publish autodocs
-      run:  bash ./publish_api_docs.sh
+      run:  bash ./generate_api_docs.sh
 
     - name: Add generated autodocs to Git repo in the documentation branch
       run: |

